### PR TITLE
Bumping latest_supported value of JwtTokenVersion to 1.2.0

### PR DESCRIPTION
**Pull request by christopappas**

### CODE REVIEW

File: edx_rest_framework_extensions/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 """ edx Django REST Framework extensions. """
 
-__version__ = '2.0.3'  # pragma: no cover
+__version__ = '2.0.4'  # pragma: no cover
```
Comment (robrap) [inline edx_rest_framework_extensions/__init__.py]: I know it is a tiny change, but I usually reserve the last for bug fixes. That would make this 2.1.0.

File: edx_rest_framework_extensions/auth/jwt/decoder.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 
 class JwtTokenVersion(object):
-    latest_supported = '1.1.0'
+    latest_supported = '1.2.0'
```
Comment (robrap) [inline edx_rest_framework_extensions/auth/jwt/decoder.py]: 1. I thought this is where we were going to comment a mini changelog of past versions, or is that somewhere in edx-platform?
2. Is this what we discussed becoming `default_latest_supported`?

File: edx_rest_framework_extensions/__init__.py (hunk 1 above)
Comment (christopappas) [reply edx_rest_framework_extensions/__init__.py]: can do

File: edx_rest_framework_extensions/auth/jwt/decoder.py (hunk 2 above)
Comment (christopappas) [reply edx_rest_framework_extensions/auth/jwt/decoder.py]: I didn't know where we intended to put the comment. I can put it here

I wasn't sure we decided to change `default_latest_supported` based on the slack messages, so I didn't. I can though

Comment (christopappas) [reply edx_rest_framework_extensions/auth/jwt/decoder.py]: Actually i'm not going to add a mini changelog. i believe we determined git log was the best place for it

**Pull request merged**